partiql-ast/src/ast.rs

```diff
@@ -17,13 +17,10 @@ use std::fmt;
 use serde::{Deserialize, Serialize};
 
 use partiql_ast_macros::Visit;
+use partiql_core::node::NodeId;
 
 pub type AstTypeMap<T> = IndexMap<NodeId, T>;
```

Comment:
Should this move and rename as well?

Reply:
If it does, then `partiql-source-map` could stop importing `ast`.

Reply:
Makes sense; could be `NodeTypeMap` or something similar.

Reply:
> Makes sense; could be `NodeTypeMap` or something similar.

Just realized I have already moved but haven't removed this one:
`pub type NodeMap<T> = IndexMap<NodeId, T>;`